docs/src/examples/multiple_shooting.md

```diff
@@ -0,0 +1,102 @@
+# Multiple Shooting
+
+In Multiple Shooting, the training data is split into overlapping intervals. The solver is then trained on individual intervals. If the end conditions of any interval co-incide with the initial conditions of the next immediate interval, then the joined/combined solution is same as solving on the whole dataset (without splitting).
```

Comment:
Line lengths

Reply:
Do you mean that I should split this into different lines? (and also preferably write shorter lines in future)

Reply:
Your text editor should show you the standard maximum line length. Break lines at that spot.